package.json

```
@@ -1,7 +1,7 @@
 {
   "license": "MIT",
   "scripts": {
-    "pretty": "prettier --write \"**/*.{js,ts,tsx,scss,html}\""
+    "pretty": "prettier --write \"**/*.{js,ts,tsx,scss,html}\";prettier --write \"output/*.json\""
```

```suggestion
    "pretty": "prettier --write \"**/*.{js,ts,tsx,scss,html,json}\""
```
and add /data/**/*.json to `.prettierignore`